portal/src/graphql/adminapi/AddEmailScreen.tsx

```diff
@@ -63,8 +64,9 @@ const AddEmailScreen: React.FC = function AddEmailScreen() {
   }, [initialFormData, formData]);
 
   const resetForm = useCallback(() => {
+    resetCreateIdentityError();
     setFormData(initialFormData);
-  }, [initialFormData]);
+  }, [initialFormData, resetCreateIdentityError]);
```

Comment:
Can we use `key` to force re-mount of the component to implement form reset?

Reply:
Currently, only `PasswordlessAuthenticatorScreen` and `PasswordsScreen` use key to force re-mount components. Maybe create new issue for updating all other screens?

Reply:
I prefer using key to re-mount the form to reset the form. It is simpler and less error-prone. Would a new abstraction (I don't know, maybe a hook) be helpful here? Maybe you guys can discuss and come up with a solution.

Reply:
For some of the screen we can use key to re-mount and reset the form, but for some other screen, the mutation hook is called on screen level, so remounting the form does not reset the error state (for example screen with tab). Then we will need to change the component structure (say introduce another wrapper element, call mutation in its child and remount its child on reset).

Reply:
Use can keep `resetCreateIdentityError()` and change `setFormData(initialFormData)` to `setRemountIdentifier((prev) => prev + 1)`.